boards/arm/npcx7m6fb_evb/Kconfig.board

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NPCX7M6FB_EVB
+	bool "Nuvoton NPCX7M6FB EVB Development board"
+	depends on SOC_NPCX7M6FB
+	help
```

Comment:
The help text is not really needed here, i think it is not common in most of arm boards